src/styles/layout/sidebar/styles.less

```diff
@@ -1,5 +1,5 @@
-// TODO: Reduce the specificity, remove the mixin declarations in favor of
-// normal styles.
+// TODO: Reduce the specificity.
+// https://mesosphere.atlassian.net/browse/DCOS-11785
```

Comment:
Could we make this just the ticket name: `DCOS-11785`? I don't think we should expose the full link